### PR TITLE
Introduce Structs to LRWholeGenomeSingleSample

### PR DESCRIPTION
**Pull request by SHuang-Broad**

### CODE REVIEW

**Comment by SHuang-Broad** [comment]
Test run id `80beac73-c930-4f9a-a668-1c5496bd580a` on methods cromwell server 36.

File: wdl/empty.options.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (kvg) [inline wdl/empty.options.json]: Can we move this to the resources/ directory?

File: wdl/get_all_dependencies.sh [hunk 2]
```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
```
Comment (kvg) [inline wdl/get_all_dependencies.sh]: This is super useful.  Can we move this to scripts/ ?

File: wdl/AlignReads.wdl [hunk 3]
```diff
@@ -1,5 +1,7 @@
 version 1.0
 
+import "Structs.wdl"
```
Comment (kvg) [inline wdl/AlignReads.wdl]: Nice - the structs make things a lot clearer.

**Pull request merged**